simpledotorg/simple-server
# Wire up follow_ups v2 in progress tab

**Story card:** [sc-3822](https://app.shortcut.com/simpledotorg/story/3822/change-follow-ups-to-track-visits-in-progress-tab)

## Because

We want to display the new follow ups v2 numbers on the progress tab, which account for a follow up via any visit as opposed to just a BP taken.

## This addresses

Retrieving the numbers from the `reporting_facility_state_groups` materialized view.

## Test instructions

Make sure `follow_ups_v2_progress_tab` and `dashboard_progress_reports` feature flags are enabled.

Then view this branch in dev or on a review app, and navigate to the progress tab report for a facility to test things out.
